tests/service/Symbols.fs

```diff
@@ -1203,3 +1203,24 @@ type T() =
     let i = 1
 """
         assertHasSymbolUsages ["i"] checkResults
+
+module Event =
+    [<Test>]
+    let ``CLIEvent member does not produce additional property symbol`` () =
+        let _, checkResults = getParseAndCheckResults """
+type T() = 
+    [<CLIEvent>]
+    member this.Event = Event<int>().Publish
+"""
+        let allSymbols =
+            checkResults.GetSymbolUsesAtLocation(4, 21, "    member this.Event = Event<int>().Publish", [ "Event" ])
+
+        let hasPropertySymbols =
+            allSymbols
+            |> List.exists (fun su ->
+                match su.Symbol with
+                | :? FSharpMemberOrFunctionOrValue as mfv -> mfv.IsProperty
+                | _ -> false
+            )
+
+        Assert.False hasPropertySymbols
```

Comment:
Hm, does it actually fix the issue? The property is probably expected there (and it should return correct `EventForFSharpProperty` symbol, I think). The unexpected is the other symbol, where the exception in `LogicalName` happens now.

Reply:
Nope, that is not the case. Your unexpected problem occurred for `this.Event`.
And it should not have existed in the first place, which is what the test covers.

Reply:
My original test was:
```
module Meh =
    [<Test>]
    let foo () =
        let _, checkResults = getParseAndCheckResults """
type T() = 
    [<CLIEvent>]
    member this.Event = Event<int>().Publish
"""
        let allSymbols = checkResults.GetAllUsesOfAllSymbolsInFile() |> Seq.toArray
        for su in allSymbols do
            match su.Symbol with
            | :? FSharpMemberOrFunctionOrValue as mfv ->
                printfn "%A" mfv.LogicalName
            | _ -> ()
```

which passed with the same code change in `CheckDeclarations.fs`.
Or course this doesn't really tell much and so I changed it to tailor to the actual problem.

Reply:
Does this work without type checking each symbol and calling `symbol.LogicalName`?

Reply:
Not sure what you mean here:
![image](https://github.com/dotnet/fsharp/assets/2621499/0cf928b2-d971-4170-b805-bd4e01ee0d44)

Reply:
Ah, OK, so it's not on the base symbol. I thought it was, thanks for checking!

Reply:
Thanks for clarifying @nojaf. Is there a way to get a single symbol generated for the event?

Reply:
`checkResults.GetSymbolUseAtLocation` would work.